### ISSUE TITLE
Train a custom 4-bar models

### ISSUE THREAD

**Issue by littlebeanhp** [allenhung1025/loop-generation#4]
Hello, thank you for this impressive work. It gave me much inspiration!

I want to train my own 4-bar model. As I dont have much music domain-knowledge, my plan is finetuning the pretrained model on a specific genre, with really low Learning rate. Can you provide me the step of resuming train step with four-bar model? Like how many seconds should we trim the datapoint, and so on...

Thank you so much
